src/sinks/mqtt/sink.rs

```diff
@@ -67,12 +69,11 @@ pub struct MqttSink {
     connector: MqttConnector,
     finalizers_queue: VecDeque<EventFinalizers>,
     quality_of_service: MqttQoS,
-    events_sent: Registered<EventsSent>,
-    bytes_sent: Registered<BytesSent>,
 }
 
 impl MqttSink {
     pub fn new(config: &MqttSinkConfig, connector: MqttConnector) -> crate::Result<Self> {
+        println!("Create sink");
```

Comment:
all the following println calls will need to be removed

Reply:
Fixed